recipes/isl/all/conanfile.py

```diff
@@ -49,7 +47,7 @@ def validate(self):
         if self.options.with_int != "gmp":
             # FIXME: missing imath recipe
             raise ConanInvalidConfiguration("imath is not (yet) available on cci")
-        if self.settings.compiler == "Visual Studio" and tools.Version(self.settings.compiler.version) < 16 and self.settings.compiler.runtime == "MDd":
+        if self.settings.compiler == "msvc" and Version(self.settings.compiler.version) < 16 and self.settings.compiler.runtime == "MDd":
```

Comment:
you have to write something like this instead:

```python
if msvc_runtime_flag(self) == "MDd":
    check_min_vs(self, "192")
```

Reply:
I believe this is addressed with the change summited below